.eslintrc.js

```diff
@@ -136,6 +136,17 @@ module.exports = {
       },
     ],
 
+    // Sort imports in a default alphabetic order
+    'sort-imports': [
+      'error', {
+        'ignoreCase': false,
+        'ignoreDeclarationSort': false,
+        'ignoreMemberSort': false,
+        'memberSyntaxSortOrder': ["none", "all", "multiple", "single"],
+        'allowSeparatedGroups': false
```

Comment:
Since these are all the defaults, do we need to specify them?

Reply:
Yes we need to, they're not in the default rule set...
I also tested on 'our' project sample after running lint:
```
   3:13  error  Member 'Trans' of the import declaration should be sorted alphabetically  sort-imports
   5:1   error  Imports should be sorted alphabetically                                   sort-imports
   6:1   error  Imports should be sorted alphabetically                                   sort-imports
   7:1   error  Expected 'multiple' syntax before 'single' syntax                         sort-imports
   8:1   error  Imports should be sorted alphabetically                                   sort-imports
  11:1   error  Imports should be sorted alphabetically                                   sort-imports
  12:1   error  Expected 'multiple' syntax before 'single' syntax                         sort-imports
  14:1   error  Imports should be sorted alphabetically                                   sort-imports
  16:1   error  Imports should be sorted alphabetically                                   sort-imports
  18:1   error  Expected 'multiple' syntax before 'single' syntax                         sort-imports
```